Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -377,6 +382,14 @@ void HTMLModelElement::applyBackgroundColor(Color color)
         m_modelPlayer->setBackgroundColor(color);
 }
 
+// MARK: - RootElement support.
+
+RefPtr<HTMLModelElementEntity> HTMLModelElement::rootEntity() const
+{
+    auto entity = HTMLModelElementEntity::create(*this);
```

Comment:
Should you really return a new one every time?

Reply:
Good catch. I probably shouldn't! Fixed it.